frontend/src/app/app.component.html

```diff
@@ -1,5 +1,6 @@
 <div>
-  <app-application-header></app-application-header>
+  <app-application-top-bar></app-application-top-bar>
+  <app-application-banner [class.backdrop]="drawerOpen"></app-application-banner>
```

Comment:
Kann zu Problemen mit dem **Content-Security-Policy** führen:
`[class.backdrop]="drawerOpen"`

Reply:
Handled in Typescript now